### PR TITLE
Implement help command

### PR DESCRIPTION
**Pull request by nadongguri**
Implemented code in help task.

### CODE REVIEW

**Comment by nadongguri** [comment]
It would be good using https://www.npmjs.com/package/gulp-task-listing also.

File: gulpfile.babel.js [hunk 1]
```diff
@@ -30,7 +30,22 @@ process.on('disconnect', () => {
 gulp.task('default', ['help']);
 
 gulp.task('help', () => {
-  // TODO(zino): We should implement this command.
+	var commands = [
```
Comment (romandev) [inline gulpfile.babel.js]: nit: indentation should be 2

Comment (romandev) [inline gulpfile.babel.js]: IMHO, it's better to use 'gulp-task-listing'.

**Comment by nadongguri** [comment]
Okay.

**Comment by Wuseok** [comment]
How about this libraries.
https://www.npmjs.com/package/gulp-help
https://www.npmjs.com/package/gulp-task-help
yesterday i used gulp-task-help library, it works fine.

**Comment by nadongguri** [comment]
Okay. Wuseok, you do it

**Comment by romandev** [comment]
any update?

**Comment by nadongguri** [comment]
@daehyunjimmy  took this...

**Comment by romandev** [comment]
Who is working on this work?

**Comment by nadongguri** [comment]
Hyena will check this old issue or discuss it with @daehyunjimmy. :-)

**Comment by nadongguri** [comment]
@ @Wuseok is doing this. #304.
Close this kona.

**Pull request closed without merging**